Fatras/include/ActsFatras/Digitization/UncorrelatedHitSmearer.hpp

```diff
@@ -24,8 +25,9 @@ namespace ActsFatras {
 /// Smearing functions definition:
 /// - it takes the unsmeared parameter
 /// - it returns the smeared parameter and a covariance
-using SmearFunction =
-    std::function<Acts::Result<std::pair<double, double>>(double)>;
+template <typename random_generator_t>
+using SmearFunction = std::function<Acts::Result<std::pair<double, double>>(
+    double, random_generator_t&)>;
```

Comment:
```suggestion
template <typename generator_t>
using SmearFunction = std::function<Acts::Result<std::pair<double, double>>(
    double, generator_t&)>;
```
Nitpicking: naming consistency with our parts of the Fatras module.

Reply:
Np, changed.